src/pypromice/core/variables/radiation.py

```diff
@@ -0,0 +1,371 @@
+__all__ = ["convert_sr", "convert_lr", "filter_lr", "filter_sr",
+           "correct_sr", "calculate_albedo"]
+
+import xarray as xr
+import numpy as np
+from pypromice.core.variables import station_pose
+
+# Define air temperature for radiometer adjustments
+T_0=273.15                  # degrees Celsius to Kelvin conversion
+deg2rad = np.pi / 180       # Degrees to radians conversion
+
+def convert_sr(sr: xr.DataArray,
```

Comment:
I'm trying to keep all docstrings for functions consistent. Let me know if there is anything else I should consider.